### PR TITLE
Simplify install and prebuild scripts

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,9 +20,9 @@
     "dev": "node-gyp rebuild",
     "fetch-libsodium": "git submodule update --recursive --init",
     "test": "standard && tape \"test/*.js\"",
-    "install": "node-gyp-build \"node preinstall.js\" \"node postinstall.js\"",
-    "prebuild": "prebuildify -a --strip --preinstall \"node preinstall.js\" --postinstall \"node postinstall.js\"",
-    "prebuild-ia32": "prebuildify -a --strip --preinstall \"node preinstall.js\" --postinstall \"node postinstall.js\" --arch=ia32"
+    "install": "node-gyp-build ./preinstall.js ./postinstall.js",
```
Comment (ralphtheninja) [inline package.json:23]: Next thing would be to mod `node-gyp-build` so that it prepends `./` if it doesn't start with `"` or `/` :smile:

Comment (emilbayes) [reply package.json:23]: Windows doesn't seem to like this

Comment (ralphtheninja) [reply package.json:23]: Oh right. It was worth a shot at least :smile:

Comment (emilbayes) [reply package.json:23]: Sure! Happy to take it if you find a fix

Comment (ralphtheninja) [reply package.json:23]: @vweevers Any idea how to make this work on windows?

Comment (vweevers) [reply package.json:23]: I guess `node-gyp-build` would have to spawn `node <arg>` in this case, or use `child_process.fork()` instead of `spawn()`.

Comment (ralphtheninja) [reply package.json:23]: Yeah, but I don't think we can assume the pre-scripts are written in node. It could be any shell script.

Comment (vweevers) [reply package.json:23]: Perhaps copy the approach of `prebuild` (which also has pre/post scripts) to `node-gyp-build`: if script extension is `.js`, then fork, else spawn.

**Pull request closed without merging**